### ISSUE TITLE
white points on the picture

### ISSUE THREAD

**Issue by thinkerthinker** [YaN9-Y/lafin#2]
hi, I find some white points on generated images. And i don't know why. 
![image](https://user-images.githubusercontent.com/37209045/70200018-3efedf00-174e-11ea-9b6c-33861aa290ce.png)
Does this happen in the pictures you generate? how to solve this problem

**Comment by YaN9-Y** [comment]
Hi thinker,
Thanks for your interest. I have also noticed this phenomenon during training. It appears to only a few samples especially those under block masks. I haven't found the exact reason yet. A further training might alleviate this in some degree.

**Comment by thinkerthinker** [comment]
thank you for u quick reply! I find spectral_norm is related to this problem. I use pytorch 0.4 for training, so the function spectral_norm is not suited for the version of my pytorch. I replace torch.nn.utils.spectral_norm with SpectralNorm(https://github.com/NVlabs/MUNIT/blob/master/networks.py). Then there are white points on generated images. But now i use torch.nn.utils.spectral_norm at pytorch1.0, white points is gone.

**Comment by jays0606** [comment]
@thinkerthinker Hi Have you found a way to fix this problem? Thank you
